src/algorithms/digi/PhotoMultiplierHitDigiConfig.h

```diff
@@ -7,19 +10,20 @@ namespace eicrecon {
     public:
 
       // random number generator seed
-      unsigned long seed = 0;
+      // FIXME: don't use 0 if `TRandomMixMax` is the RNG, it can get "stuck"
```

Comment:
How about we fail out when `seed == 0` and TRandomMixMax is used? Can we implement that? (another PR probably)

Reply:
Added a warning and changed the default to `1`.